src/ProjectService.Models.Dto/Enums/UserRoleType.cs

```diff
@@ -6,6 +6,7 @@ namespace LT.DigitalOffice.ProjectService.Models.Dto.Enums
     [JsonConverter(typeof(StringEnumConverter))]
     public enum UserRoleType
     {
-        ProjectAdmin
+        ProjectAdmin,
```

Comment:
maybe better write without Project, and add project to name of enum. Also ask @MariyaSin, what will be better User or Employee

Reply:
we talked about it, but i didn't remember we agreed to leave the admin and not rename it to the manager?

Reply:
done